packages/core/src/charts/area-stacked.ts

```diff
@@ -34,8 +35,9 @@ export class StackedAreaChart extends AxisChart {
 
 	getComponents() {
 		// Specify what to render inside the graph-frame
-		const graphFrameComponents = [
+		const graphFrameComponents: any[] = [
```

Comment:
It works, and it could be `Component[]`, but not necessary.
We just follow the interface of `getAxisChartComponents(graphFrameComponents: any[])`.
If you think this is a problem, I would recommend to create another PR for this type change.